pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -108,6 +111,14 @@ func (c *Client) Setup(t *testing.T) *Client {
 	return c
 }
 
+func (c *Client) groupVersion(group testapi.TestGroup) *unversioned.GroupVersion {
+	if c.GroupVersion != nil && group.IsEnabledVersion(c.GroupVersion) {
```

Comment:
does this mean that when I request a specific version, if its not enabled I'll get back a client that points to a different group version?